Formula/w/whatweb.rb

```diff
@@ -0,0 +1,38 @@
+class Whatweb < Formula
+  desc "Web scanner. Identify the technology stack that powers a website"
+  homepage "https://morningstarsecurity.com/research/whatweb"
+  url "https://github.com/urbanadventurer/WhatWeb/archive/refs/tags/v0.5.5.tar.gz"
+  sha256 "96dedb6a377184fb8f5fd3f2a81c26ff8c92c4dc1503ce409793a1e7ab23695d"
+  license "GPL-2.0-only"
+  head "https://github.com/urbanadventurer/WhatWeb.git", branch: "master"
+
+  depends_on "rbenv"
+  depends_on "ruby-build"
```

Comment:
Why?

Reply:
I don't have a good reason, i thought it was required. Removed now.